### PR TITLE
docs: Upgrade instructions for 0.17

### PR DESCRIPTION
**Pull request by mowies**
### This PR
- adds instructions on how to upgrade from Keptn 0.16.x to 0.17.0

### CODE REVIEW

File: content/docs/install/upgrade/index.md [hunk 1]
```diff
@@ -6,17 +6,60 @@ weight: 500
...
-you can also deploy the *resource-service* and configure the Git repositories later. For this, a backup is necessary.
-
-1. Back up the [configuration-service](../../0.17.x/operate/backup_and_restore/#back-up-configuration-service).
-2. For each Keptn project in the backup data, open a shell in that directory and make sure the `Git` CLI is available.
-3. Attach your upstream to the Keptn project via the Git CLI with `git remote add origin <remoteURL>`, where `<remoteURL>` is your Git upstream.
-4. Run `git push --all` to synchronize your backup with your Git repository.
-5. Upgrade to Keptn 0.17.x using `helm upgrade keptn keptn/keptn -n keptn --version 0.17.0 --reuse-values`
-6. Navigate to your Bridge installation and configure an upstream to the Keptn projects.
+With Keptn 0.17.x, the Keptn CLI commands for `install`, `uninstall` and `upgrade` were deprecated. We recommend to upgrade using the Helm CLI instead.
+Additionally, the Keptn Helm chart was refactored heavily to make it ready for future features and structural changes.
```
Comment (StackScribe) [inline content/docs/install/upgrade/index.md:10]:
```suggestion
Additionally, the Keptn Helm chart has  been refactored heavily to make it ready for future features and structural changes.
```

File: content/docs/install/upgrade/index.md [hunk 2]
```diff
@@ -6,17 +6,60 @@ weight: 500
...
+To upgrade, we recommend getting your old Helm values file, and then merging it together with the default Keptn Helm values
+to get any new default values that were introduced. Afterwards, you can use your new values file to follow the
+[Helm installation instructions](../install/#install-keptn).
```
Comment (StackScribe) [inline content/docs/install/upgrade/index.md:13]:
```suggestion
[Helm installation instructions](../install/#helm-install).
```

File: content/docs/install/upgrade/index.md [hunk 3]
```diff
@@ -6,17 +6,60 @@ weight: 500
...
+
+Detailed step by step guide:
+- Follow [Keptn's backup instructions](../backup_and_restore)
```
Comment (StackScribe) [inline content/docs/install/upgrade/index.md:16]:
```suggestion
- Follow [Keptn's backup instructions](../../0.17.x/operate/backup_and_restore)
```

File: content/docs/install/upgrade/index.md [hunk 4]
```diff
@@ -6,17 +6,60 @@ weight: 500
...
+- Make sure you are connected to the kubernetes cluster where Keptn is installed.
```
Comment (StackScribe) [inline content/docs/install/upgrade/index.md:17]:
```suggestion
- Make sure you are connected to the Kubernetes cluster where Keptn is installed.
```

Comment (StackScribe) [inline content/docs/install/upgrade/index.md:17]:
```suggestion
- Make sure you are connected to the Kubernetes cluster where Keptn is installed.
```

Comment (StackScribe) [inline content/docs/install/upgrade/index.md:17]:
```suggestion
- Make sure you are connected to the Kubernetes cluster where Keptn is installed.
```

File: content/docs/install/upgrade/index.md [hunk 5]
```diff
@@ -6,17 +6,60 @@ weight: 500
...
+- Fetch your current Helm values with `helm get values -n <your-keptn-namespace> <your-keptn-release-name> > keptn-values.yaml`
+   For namespace `keptn-test` and release name `keptn` (the default release name) the command would look like this:
```
Comment (StackScribe) [inline content/docs/install/upgrade/index.md:19]:
```suggestion
   For namespace `keptn-test` and release name `keptn` (the default release name), the command would look like this:
```

File: content/docs/install/upgrade/index.md [hunk 6]
```diff
@@ -6,17 +6,60 @@ weight: 500
...
+    e.g. `common.strategy.type` -> `strategy.type`
+- After adjusting your Helm values you are ready to upgrade to the new version of Keptn. Since the `keptn upgrade` CLI command
+   is deprecated with Keptn 0.17, please use Helm directly to do the upgrade:
+
+   ```
+   helm upgrade keptn keptn/keptn -n keptn-test --version 0.17.0 --values <your-adjusted-values-file>
+   ```
+
+If you have helm-service or jmeter-service installed, please follow the steps below to upgrade:
+- Make sure you are connected to the kubernetes cluster where Keptn is installed.
```
Comment (StackScribe) [inline content/docs/install/upgrade/index.md:41]:
```suggestion
- Make sure you are connected to the Kubernetes cluster where Keptn is installed.
```

File: content/docs/install/upgrade/index.md [hunk 7]
```diff
@@ -6,17 +6,60 @@ weight: 500
...
+- Fetch your current Helm values with `helm get values -n <your-exec-plane-namespace> <your-exec-plane-service-release-name> > old-values.yaml`
+  For namespace `exec-plane` and release name `helm-service` (the default release name) the command would look like this:
```
Comment (StackScribe) [inline content/docs/install/upgrade/index.md:43]:
```suggestion
  For namespace `exec-plane` and release name `helm-service` (the default release name), the command would look like this:
```

**Review by StackScribe — CHANGES_REQUESTED** [review]
I fixed a couple broken xrefs and made some editorial changes for your consideration.  The xrefs need to be fixed or the PR doesn't pass CI ;-). The others are optional.

**Review by oleg-nenashev — APPROVED** [review]
LGTM

File: content/docs/install/access/index.md [hunk 8]
```diff
@@ -25,9 +25,9 @@ width="1000px">}}
 This option exposes Keptn externally using a cloud provider's load balancer (if available).
 
 1. **Install Keptn:** For installing Keptn on your cluster, please use the Helm CLI.
-Depending on whether you would like to install the execution plane for continuous delivery, add the flag `continuous-delivery.enabled=true`.
+Depending on whether you would like to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
```
Comment (StackScribe) [inline content/docs/install/access/index.md:28]:
```suggestion
If you want to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
```

File: content/docs/install/access/index.md [hunk 9]
```diff
@@ -25,9 +25,9 @@ width="1000px">}}
 This option exposes Keptn externally using a cloud provider's load balancer (if available).
 
 1. **Install Keptn:** For installing Keptn on your cluster, please use the Helm CLI.
```
Comment (StackScribe) [inline content/docs/install/access/index.md:27]:
```suggestion
1. **Install Keptn:** Use the Helm CLI to install Keptn on your cluster.
```

File: content/docs/install/access/index.md [hunk 10]
```diff
@@ -57,9 +57,9 @@ Depending on whether you would like to install the execution plane for continuou
 This option exposes Keptn on each Kubernetes Node's IP at a static port. Therefore, please make sure that you can access the Kubernetes Nodes in your network.
 
 1. **Install Keptn:** For installing Keptn on your cluster, please use the Keptn CLI.
```
Comment (StackScribe) [inline content/docs/install/access/index.md:59]:
```suggestion
1. **Install Keptn:** Use the Keptn CLI to install Keptn on your cluster.
```

File: content/docs/install/access/index.md [hunk 11]
```diff
@@ -57,9 +57,9 @@ Depending on whether you would like to install the execution plane for continuou
...
-Depending on whether you would like to install the execution plane for continuous delivery, add the flag `continuous-delivery.enabled=true`.
+Depending on whether you would like to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
```
Comment (StackScribe) [inline content/docs/install/access/index.md:60]:
```suggestion
If you want to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
```

File: content/docs/install/access/index.md [hunk 12]
```diff
@@ -93,9 +93,9 @@ Depending on whether you would like to install the execution plane for continuou
 ### Option 3: Expose Keptn via an Ingress
 
 1. **Install Keptn:** For installing Keptn on your cluster, please use the Keptn CLI.
```
Comment (StackScribe) [inline content/docs/install/access/index.md:95]:
```suggestion
1. **Install Keptn:** Use the Keptn CLI to install Keptn on your cluster.
```

File: content/docs/install/access/index.md [hunk 13]
```diff
@@ -93,9 +93,9 @@ Depending on whether you would like to install the execution plane for continuou
...
-Depending on whether you would like to install the execution plane for continuous delivery, add the flag `continuous-delivery.enabled=true`.
+Depending on whether you would like to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
```
Comment (StackScribe) [inline content/docs/install/access/index.md:96]:
```suggestion
If you want to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
```

File: content/docs/install/access/index.md [hunk 14]
```diff
@@ -293,9 +293,9 @@ Depending on whether you would like to install the execution plane for continuou
 This option does not expose Keptn to the public but exposes Keptn on a *cluster-internal* IP.
 
 1. **Install Keptn:** For installing Keptn on your cluster, please use the Keptn CLI.
```
Comment (StackScribe) [inline content/docs/install/access/index.md:295]:
```suggestion
1. **Install Keptn:** Use the Keptn CLI to install Keptn on your cluster.
```

File: content/docs/install/access/index.md [hunk 15]
```diff
@@ -293,9 +293,9 @@ Depending on whether you would like to install the execution plane for continuou
...
-Depending on whether you would like to install the execution plane for continuous delivery, add the flag `continuous-delivery.enabled=true`.
+Depending on whether you would like to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
```
Comment (StackScribe) [inline content/docs/install/access/index.md:296]:
```suggestion
If you want to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
```

File: content/docs/install/upgrade/index.md [hunk 16]
```diff
@@ -6,17 +6,62 @@ weight: 500
...
-Suppose you need the additional features provided by the *resource-service*,  such as HTTPS/SSH or Proxy, to configure your Keptn project with an upstream. In that case,
-you can also deploy the *resource-service* and configure the Git repositories later. For this, a backup is necessary.
-
-1. Back up the [configuration-service](../../0.17.x/operate/backup_and_restore/#back-up-configuration-service).
-2. For each Keptn project in the backup data, open a shell in that directory and make sure the `Git` CLI is available.
-3. Attach your upstream to the Keptn project via the Git CLI with `git remote add origin <remoteURL>`, where `<remoteURL>` is your Git upstream.
-4. Run `git push --all` to synchronize your backup with your Git repository.
-5. Upgrade to Keptn 0.17.x using `helm upgrade keptn keptn/keptn -n keptn --version 0.17.0 --reuse-values`
-6. Navigate to your Bridge installation and configure an upstream to the Keptn projects.
+With Keptn 0.17.x, the Keptn CLI commands for `install`, `uninstall` and `upgrade` were deprecated. We recommend to upgrade using the Helm CLI instead.
```
Comment (StackScribe) [inline content/docs/install/upgrade/index.md:9]:
```suggestion
With Keptn 0.17.x, the Keptn CLI commands for `install`, `uninstall` and `upgrade` were deprecated. We recommend that you instead use the Helm CLI to upgrade.
```
Is this really a recommendation, implying that they have other options?  Maybe it should say, "Instead, use the Helm CLI to upgrade."

File: content/docs/install/manual-install/index.md [hunk 17]
```diff
@@ -25,7 +25,7 @@ and [Automated Operations](../../concepts/automated_operations/) use cases. To i
     ```
     helm repo add keptn https://charts.keptn.sh
     helm repo update
-    helm install keptn keptn/keptn -n keptn --version=$KeptnVersion --create-namespace --set=continuous-delivery.enabled=true
+    helm install keptn keptn/keptn -n keptn --version=$KeptnVersion --create-namespace --set=continuousDelivery.enabled=true
```
Comment (StackScribe) [inline content/docs/install/manual-install/index.md:28]: Actually, I think this file will be deleted by https://github.com/keptn/keptn.github.io/pull/1287 .  Is that correct?  I guess we can look at everything after we merge everything and adjust accordingly

**Review by StackScribe — APPROVED** [review]
Looks good!  I suggested a couple of prose-tightening edits that you may use or discard but this is a nice piece of work!

**Pull request merged**